app/code/Magento/Sales/Block/Adminhtml/Order/View/Tab/History.php

```diff
@@ -76,6 +76,7 @@ public function getFullHistory()
         $history = [];
         foreach ($order->getAllStatusHistory() as $orderComment) {
             $history[] = $this->_prepareHistoryItem(
+                $orderComment->getId(),
```

Comment:
Good catch on this.  Looking back on the patch I'm running I did create a new method for `_prepareHistoryItem` since I added the ID.  I missed that when making this PR.  Look for that in an upcoming change.